server/services/lists/caCaseloadService.ts

```diff
@@ -59,137 +63,188 @@ export type ManagedCase = {
-          .filter(c =>
-            [
-              LicenceStatus.NOT_STARTED,
-              LicenceStatus.TIMED_OUT,
-              LicenceStatus.NOT_IN_PILOT,
-              LicenceStatus.OOS_RECALL,
-              LicenceStatus.OOS_BOTUS,
-            ].some(status => c.licences.find(l => l.status === status))
-          )
-      })
+    if (!prisonersApproachingRelease.length) {
+      return []
+    }
 
-    const [withLicence, pending] = await Promise.all([casesWithLicences, casesPendingLicence])
-    const casesWithComs = await this.mapResponsibleComsToCases(withLicence.concat(pending))
+    const prisonersWithoutLicences = prisonersApproachingRelease.filter(
+      p => !licenceNomisIds.includes(p.prisoner.prisonerNumber)
+    )
+
+    if (!prisonersWithoutLicences.length) {
+      return []
+    }
 
-    return casesWithComs
+    const eligiblePrisoners = await this.filterOffendersEligibleForLicence(prisonersWithoutLicences, user)
+    const casesWithoutLicences = await this.pairNomisRecordsWithDelius(eligiblePrisoners)
+    return this.createNotStartedLicenceForCase(casesWithoutLicences)
   }
 
-  private caCaseload(cases: Case[], searchString: string, view: 'prison' | 'probation'): CaCaseLoad {
-    const caseLoad = this.caseloadViewModel(cases)
-    const showAttentionNeededTab = caseLoad.some(e => e.tabType === CaViewCasesTab.ATTENTION_NEEDED)
-    const searchResults = this.applySearch(searchString, caseLoad)
-    const sortResults = this.applySort(searchResults, view)
-    return { cases: sortResults, showAttentionNeededTab }
+  private enrichWithNomisData(licences: LicenceSummary[], prisoners: CaseloadItem[]): CaCase[] {
+    return prisoners
+      .map(p => {
+        const licencesForOffender = licences.filter(l => l.nomisId === p.prisoner.prisonerNumber)
+        const licence = this.findLatestLicenceSummary(licencesForOffender)
+        const releaseDate = parseCvlDate(licence.actualReleaseDate || licence.conditionalReleaseDate)
+        return {
+          kind: <LicenceKind>licence.kind,
+          licenceId: licence.licenceId,
+          licenceVersionOf: licence.versionOf,
+          name: convertToTitleCase(`${licence.forename} ${licence.surname}`.trim()),
+          prisonerNumber: licence.nomisId,
+          releaseDate: releaseDate ? format(releaseDate, 'dd MMM yyyy') : 'not found',
+          releaseDateLabel: licence.actualReleaseDate ? 'Confirmed release date' : 'CRD',
+          licenceStatus: <LicenceStatus>licence.licenceStatus,
+          nomisLegalStatus: p.prisoner.legalStatus,
+          lastWorkedOnBy: licence.updatedByFullName,
+          isDueForEarlyRelease: licence.isDueForEarlyRelease,
+          isInHardStopPeriod: licence.isInHardStopPeriod,
+          tabType: determineCaViewCasesTab(p.prisoner, p.cvl, licence),
+          probationPractitioner: {
+            staffUsername: licence.comUsername,
+          },
+        }
+      })
+      .filter(c => c)
```

Comment:
**question:** `c` is never going to be falsy is it?

Reply:
I'm pretty sure you're right and it's left over from a previous iteration
Will confirm and remove it if so!